docs/components/COMPONENT_REPLACEMENT.md

```diff
@@ -1,22 +1,21 @@
 # Component replacement
```

Comment:
There are many mentions of customization app/base app but we never introduce the concepts and we do not provide the script apply custo. We should probably rewrite this example and provide a simple example with a fetch of the replacement map